composer.json

```diff
@@ -25,7 +25,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Craftgate\\": "lib/Craftgate/"
+            "Craftgate\\": "src/"
```

Comment:
src altına neden taşındığını açıklar mısınız?

Reply:
PSR ekosisteminde `namespace` => `src` yapısı yangın olduğu için bu şekilde uygundur diye düşünmüştüm.

Reply:
👍